Makefile

```diff
@@ -72,9 +72,6 @@ AUR_RELEASE_OPTIONS ?=
 .PHONY: release
 release: test
 	[ -n "$(VERSION)" ]
-	sed -E "s/^(DOWNGRADE_VERSION=)(.*)/\1\"v$(VERSION)\"/g" -i downgrade
-	git add downgrade
-	git commit -m "Release v$(VERSION)"
```

Comment:
Oh, I see how this works now. I'd recommend we remove the need to specify `VERSION` for `make`, and use something like:

```make
VERSION ?= $(shell sed -E /^DOWNGRADE_VERSION=(.*)/!d; s//\1/' downgrade)
```

That way, the version is in exactly one place (the script).

Reply:
That works, but it carries the quotes as well. How about this?

```shell
sed -E '/^DOWNGRADE_VERSION=(.*)/!d; s//\1/' downgrade | tr -d "'" | tr -d '"'
```

Reply:
I tried removing the quotes within `sed` but I keep getting syntax errors because of nested double and single quotes. This option worked easily.

Reply:
Well I think we have tight control on this variable, so we can either not quote it, or quote it consistently and expect that in the regex, no need to be flexible withe single-double, `tr`.

```sh
sed '^DOWNGRADE_VERSION="\([^"]*\)".*$/!d; s//\1/'
```

In case you didn't realize, using `\(` and `\)` in this regex means you won't need `-E`. Since it's such a minor thing, I tend to do that to avoid the GNU-only flag.

Reply:
Makes sense, thanks